### PR TITLE
Enhance converters with allow serialize empty

### PR DESCRIPTION
**Pull request by healerz**

### CODE REVIEW

**Comment by dominikzogg** [comment]
please replace 3.2 with 3.3:
https://github.com/saxulum/saxulum-elasticsearch-querybuilder/blob/master/composer.json#L24
https://github.com/saxulum/saxulum-elasticsearch-querybuilder/blob/master/README.md#installation

please add doc:
https://github.com/saxulum/saxulum-elasticsearch-querybuilder/blob/master/doc/Converter/IteratableToNodeConverter.md
https://github.com/saxulum/saxulum-elasticsearch-querybuilder/blob/master/doc/Converter/ScalarToNodeConverter.md

**Review by dominikzogg — CHANGES_REQUESTED** [review]
please replace 3.2 with 3.3:
https://github.com/saxulum/saxulum-elasticsearch-querybuilder/blob/master/composer.json#L24
https://github.com/saxulum/saxulum-elasticsearch-querybuilder/blob/master/README.md#installation

please add doc:
https://github.com/saxulum/saxulum-elasticsearch-querybuilder/blob/master/doc/Converter/IteratableToNodeConverter.md
https://github.com/saxulum/saxulum-elasticsearch-querybuilder/blob/master/doc/Converter/ScalarToNodeConverter.md

**Comment by healerz** [comment]
@dominikzogg  ok like this? :)

File: doc/Converter/IteratableToNodeConverter.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # IteratableToNodeConverter
 
+**Important**: By default empty nodes get not serialized. NullNode forces null value serialization.
```
Comment (dominikzogg) [inline doc/Converter/IteratableToNodeConverter.md]: I mean't another sample with the new argument

**Review by dominikzogg — APPROVED** [review]

**Comment by dominikzogg** [comment]
@healerz thanks

**Pull request merged**